### PR TITLE
Update to wpad

### PR DESCRIPTION
**Pull request by mimugmail**
If you read the guide the first time it's a bit confusing regarding negate etc. 
With this small sentence it clear what we want to achieve and this fits 90% of all usage how this is used.

### CODE REVIEW

**Comment by fichtner** [comment]
@fabianfrz please review :)

**Comment by fichtner** [comment]
ah, now I can assign you. better...

File: source/manual/how-tos/pac.rst [hunk 1]
```diff
@@ -40,7 +40,9 @@ Go to 'Services' -> Proxy -> Configuration and open Match
 .. image:: images/pac_menu_match.png
 
 Enter a name for the match and an optional description.
-Then choose what to match on.
+Then choose what to match on. In our example we want to achieve
+that hostnames without a FQDN (like SERVER01) and also our
```
Comment (fabianfrz) [inline source/manual/how-tos/pac.rst]: would not use caps for the hostname.

**Comment by mimugmail** [comment]
changed to lower :)

**Comment by fichtner** [comment]
Merged, thanks!

**Comment by fabianfrz** [comment]
@mimugmail uppercase hostnames are are not common on Linux and BSD ;)

**Pull request merged**